distribution/docker/build.gradle

```diff
@@ -165,6 +168,30 @@ tasks.register("copyKeystore", Sync) {
   }
 }
 
+tasks.register("ensureLog4jConfigurationSync") {
```

Comment:
I wonder if we should wire this to `precommit` so it fails early. Otherwise this will only fail when we actually go to build these docker images.